### PR TITLE
PHOENIX-7175 : Set java.io.tmpdir to the maven build directory for tests

### PR DESCRIPTION
**Pull request by Divneet18**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -621,6 +622,9 @@
           <shutdown>exit</shutdown>
           <trimStackTrace>false</trimStackTrace>
           <forkNode implementation="org.apache.maven.plugin.surefire.extensions.SurefireForkNodeFactory"/>
+          <systemPropertyVariables>
```
Comment (stoty) [inline pom.xml:625]: please add the same to the common configuration section of the failsafe plugin.
Hbase doesn't use failsafe, but Phoenix does.

**Review by stoty — CHANGES_REQUESTED** [review]

**Review by stoty — APPROVED** [review]
+1 LGTM

**Comment by stoty** [comment]
merged manually

**Comment by ranganathg** [comment]
> merged manually

what does merge manually mean? Why not use the regular merge?

**Comment by stoty** [comment]
via git cli, not github.
I add this to avoid confusion when the PR is closed.

**Pull request closed without merging**